include/pops/network.hpp

```diff
@@ -680,6 +724,42 @@ class Network
         }
     }
 
+    /**
+     * @brief Convert string to probability
+     *
+     * @param text String with probability
+     * @return Probability as number
+     */
+    static double probability_from_text(const std::string& text)
+    {
+        try {
+            return std::stod(text);
```

Comment:
Good point. 04744f9 adds the check and tests for the good and bad cases.

Reply:
Since f2d11d54a86a011c6b51f390e97c894c0230aa27 only `>1` is enforced. Any other range is allowed and meaning is up to the user thanks to the underlying std::discrete_distribution function.